src/app/query/bitcoin/ordinals/use-inscriptions.query.ts

```diff
@@ -52,13 +60,21 @@ export function useInscriptionsInfiniteQuery() {
   // TO-DO remove code for testing purposes before merge
   const searchParams = new URLSearchParams(document.location.search);
   const testInscriptionAddress = searchParams.get('testAddress');
-  const currentNumberOfAddressesWithoutOrdinals = useRef(0);
 
-  const fromIndex = useRef(0);
-  const addressesMap = useRef(
-    getAddressesData(fromIndex.current, fromIndex.current + addressesSimultaneousFetchLimit)
+  const currentAccData = useRef<CurrentAccData>({});
```

Comment:
I wonder if this actually a use case for `new Map<number, AccountData>()`, esp. considering properties can't begin with number unless accessed as string `x.0 // unexpected number`

Can you remind me why we need to do the map of address again?

Reply:
hm, I realise now that we need is only to refresh this `currentAccData` once account has been changed and that's it, we don't need map here

Reply:
done, without map